VSGitDiff/GitDiffPackage.cs

```diff
@@ -6,13 +6,13 @@
 
 namespace VSGitDiff
 {
-    [PackageRegistration(UseManagedResourcesOnly = true)]
+    [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
```

Comment:
Changes for this file to support asynchronous auto loading and use of JoinableTaskFactory.SwitchToMainThreadAsync in InitializeAsync is based on [this MS GitHub repo on migrating to AsyncPackage](https://github.com/Microsoft/VSSDK-Extensibility-Samples/tree/master/AsyncPackageMigration).